manifests/y/Yemiao/ZhiyunTranslator/7.7.3/Yemiao.ZhiyunTranslator.installer.yaml

```
@@ -0,0 +1,19 @@
+# Created with YamlCreate.ps1 v2.0.7 using InputObject 🤖 $debug=QUSU-7-2-3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Yemiao.ZhiyunTranslator
+PackageVersion: 7.7.3
+MinimumOSVersion: 10.0.0.0
+InstallerType: inno
+Scope: machine
+UpgradeBehavior: install
+FileExtensions:
+- pdf
+ReleaseDate: 2022-05-08
+Installers:
+- Architecture: x86
+  InstallerUrl: https://cloud.06dn.com/api/v3/file/source/128835/ZhiyunTrans_setup7.7.3A-down.exe
```

```suggestion
	InstallerUrl: https://note.youdao.com/yws/api/personal/file/WEB74f25b2dad280e152b81582e4ada67f3?method=download&inline=true&shareKey=cdfe866a8850ddddfe23037089d51ad0
```
From https://free.zhiyunwenxian.cn/zy/UpdateURL.txt